From 5a49344fb6f36d1b2f4c95493f7ef1171105add8 Mon Sep 17 00:00:00 2001
From: Akshay Gupta <Akshay.Gupta@amd.com>
Date: Tue, 7 Apr 2026 13:13:01 +0530
Subject: [PATCH 4/4] jtag: jtag-aspeed: Increase the wait iteration to 300

- Increase the wait iteration from 75 to 300 as per Aspeed changes
---
 drivers/jtag/jtag-aspeed.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/jtag/jtag-aspeed.c b/drivers/jtag/jtag-aspeed.c
index a8959096e..4409b11a6 100644
--- a/drivers/jtag/jtag-aspeed.c
+++ b/drivers/jtag/jtag-aspeed.c
@@ -107,7 +107,7 @@
 
 #define ASPEED_JTAG_TCK_WAIT		10
 #define ASPEED_JTAG_RESET_CNTR		10
-#define WAIT_ITERATIONS		75
+#define WAIT_ITERATIONS		300
 
 /*#define USE_INTERRUPTS*/
 
-- 
2.34.1

